examples/dialog-modal/datepicker-dialog.html

```diff
@@ -0,0 +1,739 @@
+            <td>
+              <ul>
+                <li>Change the grid of dates to the next month.</li>
+                <li>The focus will be on the same day of that week, if it does not exist, then it will move the focus to the same day of previous or next week</li>
+              </ul>
+            </td>
+          </tr>
+          <tr data-test-id="grid-shift-pagedown">
+            <th><kbd>Shift</kbd>+<br> <kbd>PageDown</kbd></th>
+            <td>
+              <ul>
+                <li>Change the grid of dates to the next Year.</li>
+                <li>The focus will be on the same day of that week, if it does not exist, then it will move the focus to the same day of previous or next week</li>
+              </ul>
+            </td>
+          </tr>
+          <tr data-test-id="dialog-tab">
+            <th><kbd>TAB</kbd></th>
+            <td>Move focus to next button inside the dialog.</td>
+          </tr>
+          <tr data-test-id="dialog-shift-tab">
+            <th><kbd>Shift</kbd> +<br> <kbd>TAB</kbd></th>
+            <td>Move focus to previous button inside the dialog.</td>
+          </tr>
+          <tr data-test-id="dialog-esc">
+            <th><kbd>ESC</kbd></th>
+            <td>
+              <ul>
+                <li>Close the dialog and move focus back to calendar button.</li>
+                <li>The value of the date texbox is not updated.</li>
+              </ul>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </section>
+
+    <section>
+        <h3 id="kbd_label_6">Date Picker Buttons (OK and Cancel)</h3>
+        <table aria-labelledby="kbd_label_6 kbd_label" class="def">
+          <thead>
+            <tr>
+                <th>Key</th>
+                <th>Function</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr data-test-id="okay-cancel-button-space-return">
+              <th> <kbd>Space</kbd>,<br> <kbd>Return</kbd></th>
+              <td>
+                <ul>
+                    <li>The Cancel button closes the date picker dialog, moves focus to calendar button, does not update date in date input.</li>
+                    <li>The OK button closes the date picker dialog, moves focus to calendar button, does update date in date input.</li>
+                </ul>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+    </section>
+  </section>
+
+  <section>
+    <h2 id="rps_label">Role, Property, State, and Tabindex  Attributes</h2>
+
+
+    <section>
+        <h3 id="rps_label_2">Choose Date Button</h3>
+        <table aria-labelledby="rps_label_2 rps_label" class="data attributes">
+          <thead>
+            <tr>
+              <th scope="col">Role</th>
+              <th scope="col">Attribute</th>
+              <th scope="col">Element</th>
+              <th scope="col">Usage</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr data-test-id="test-not-required">
```

Comment:
Please delete this row. We do not document implied roles.

Reply:
@mcking65 
Removed the implied button role